client/app/routes/notifications_.$id.tsx

```diff
@@ -0,0 +1,25 @@
+import { LoaderFunctionArgs, json, redirect } from "@remix-run/node";
+import { checkAuth } from "~/lib/check-auth";
+import { prisma } from "~/lib/prisma.server";
+
+export const loader = async ({ request, params }: LoaderFunctionArgs) => {
+	const userId = await checkAuth(request);
+
+	const notificationId = Number(params.id as string);
+
+	const notification = await prisma.notification.findFirst({
+		where: { id: notificationId },
+	});
+
+	if (notification) {
+		switch (notification.entityType) {
+			case "post":
+				await prisma.notificationSuscribers.updateMany({
```

Comment:
It can't only be one.

```suggestion
				await prisma.notificationSuscribers.updateOne({
```

Reply:
prisma update  method supports only where is via primary key but this update is via notificationId and userId